apps/api/v2/src/modules/selected-calendars/selected-calendars.repository.ts

```diff
@@ -6,7 +6,7 @@ import { Injectable } from "@nestjs/common";
 export class SelectedCalendarsRepository {
   constructor(private readonly dbRead: PrismaReadService, private readonly dbWrite: PrismaWriteService) {}
 
-  createSelectedCalendar(externalId: string, credentialId: number, userId: number, integration: string) {
+  persistSelectedCalendar(externalId: string, credentialId: number, userId: number, integration: string) {
```

Comment:
I would call it `upsertSelectedCalendar` for clarity because persist could also mean just to create but thats just my opinion.